### DISCUSSION TITLE
update preset docs: Creating preset now requires name: https://panda-css.com/docs/customization/presets#creating-a-preset

### DISCUSSION THREAD

**Discussion by omridevk** [chakra-ui/panda#2751] [General]
### Description

We should update the docs to reflect to the new required property

### Link to Reproduction

https://stackblitz.com/edit/vitejs-vite-lfwyue?file=src%2FApp.tsx&terminal=dev

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Panda CSS Version

latest

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by anubra266** [comment]
@omridevk feel free to open a PR

**Reply by segunadebayo** [reply]
Pushed an update for this.

**Reply by omridevk** [reply]
> @omridevk feel free to open a PR

Sorry didnt have the time
